app/controllers/support/PartListFilterSupport.scala

```diff
@@ -0,0 +1,17 @@
+package controllers.support
+
+import play.api.data.Form
+import play.api.data.Forms._
+import scala.beans.BeanProperty
+
+trait PartListFilterSupport {
+
+  val partListFilterForm: Form[PartListFilter] = Form(
+    mapping(
+      "ids" -> seq(nonEmptyText)
+    )(PartListFilter.apply)(PartListFilter.unapply)
+  )
+
+}
+
+case class PartListFilter(@BeanProperty ids: Seq[String])
```

Comment:
I thought it was required for Swagger to pick up the param, but it seems to work without. Removed.